inc/classes/class-srm-redirect.php

```diff
@@ -285,7 +285,7 @@ public function maybe_redirect() {
 		 * @param {string} $request_path Request path. Default `$_SERVER['REQUEST_URI']`.
 		 * @returns {string} Request path.
 		 */
-		$requested_path   = esc_url_raw( apply_filters( 'srm_requested_path', $_SERVER['REQUEST_URI'] ) );
+		$requested_path   = esc_url_raw( apply_filters( 'srm_requested_path', $_SERVER['REQUEST_URI'] ?? '' ) );
```

Comment:
@jspellman814, can we also sanitize this value?

Reply:
After this patch, I am seeing the following warning - 

<img width="793" alt="image" src="https://github.com/10up/safe-redirect-manager/assets/1784821/12da8b87-c376-4725-b745-fd4381413dd6">

Reply:
@ravinderk I pushed a fix for this in [07db92f](https://github.com/10up/safe-redirect-manager/pull/356/commits/07db92fa6406daf15612e84b3a4817cb725bbad7).